README.md

```diff
@@ -1,17 +1,17 @@
 # Rust Space API Implementation
 
-[![Build Status](https://img.shields.io/travis/coredump-ch/spaceapi-server-rs/master.svg)](https://travis-ci.org/coredump-ch/spaceapi-server-rs)
-[![Test Coverage](https://img.shields.io/coveralls/coredump-ch/spaceapi-server-rs/master.svg)](https://coveralls.io/github/coredump-ch/spaceapi-server-rs)
+[![Build Status](https://img.shields.io/travis/spaceapi-community/spaceapi-server-rs/master.svg)](https://travis-ci.org/spaceapi-community/spaceapi-server-rs)
+[![Test Coverage](https://img.shields.io/coveralls/spaceapi-community/spaceapi-server-rs/master.svg)](https://coveralls.io/github/spaceapi-community/spaceapi-server-rs)
 [![Crates.io Version](https://img.shields.io/crates/v/spaceapi-server.svg)](https://crates.io/crates/spaceapi-server)
 [![Crates.io Downloads](https://img.shields.io/crates/d/spaceapi-server.svg)](https://crates.io/crates/spaceapi-server)
-[![Dependency Status](https://dependencyci.com/github/coredump-ch/spaceapi-server-rs/badge)](https://dependencyci.com/github/coredump-ch/spaceapi-server-rs)
-[![Clippy Linting Result](https://clippy.bashy.io/github/coredump-ch/spaceapi-server-rs/master/badge.svg)](https://clippy.bashy.io/github/coredump-ch/spaceapi-server-rs/master/log)
+[![Dependency Status](https://dependencyci.com/github/spaceapi-community/spaceapi-server-rs/badge)](https://dependencyci.com/github/spaceapi-community/spaceapi-server-rs)
+[![Clippy Linting Result](https://clippy.bashy.io/github/spaceapi-community/spaceapi-server-rs/master/badge.svg)](https://clippy.bashy.io/github/spaceapi-community/spaceapi-server-rs/master/log)
 
 This is a library that allows an easy implementation of a
 [SpaceAPI](http://spaceapi.net/) v0.13 server in Rust.
 
-- Crate Documentation: https://coredump-ch.github.io/spaceapi-server-rs/
-- Space API Reference: http://spaceapi.net/documentation
+- Crate Documentation: https://spaceapi-community.github.io/spaceapi-server-rs/
```

Comment:
Shouldn't we link to docs.rs instead?

Reply:
I think you once made the argument, that docs.rs is only for the released version and the link here provides up do date documentation for the master version.

Reply:
Could be, but at least now I think that the master docs aren't really relevant to the users, and the developers can just look at the code or build the docs locally :)

Reply:
This way we could also remove the docs stuff from the Travis build.

Reply:
And btw, the repository is already multiple megabytes, because all the docs are pushed into a branch 😕

Reply:
> And btw, the repository is already 64 MiB(!!), because all the docs are pushed into a branch 😕

Where do you see that? Locally it's only 8.9MiB for me:

    % du -h  --max-depth=0 .git   
    8.9M	.git
Maybe you need to run `git gc`

Reply:
Fixed.